challenge-manager/edge-tracker/tracker.go

```diff
@@ -329,16 +347,62 @@ func (et *Tracker) Act(ctx context.Context) error {
 			return et.fsm.Do(edgeAwaitConfirmation{})
 		}
 		return et.fsm.Do(edgeConfirm{})
-	case edgeConfirmed:
+	case EdgeConfirmed:
 		srvlog.Info("Edge reached confirmed state", fields)
 		return et.fsm.Do(edgeConfirm{})
 	default:
 		return fmt.Errorf("invalid state: %s", current.State)
 	}
 }
 
-func (et *Tracker) shouldComplete() bool {
-	return et.fsm.Current().State == edgeConfirmed
+// Checks if an edge tracker should despawn and no longer act.
```

Comment:
Proper godoc comment?

```suggestion
// ShouldDespawn checks if an edge tracker should despawn and no longer act.
```